infra/ansible/playbooks/k8s/k8s-hosts-prod

```diff
@@ -0,0 +1,16 @@
+k8s_cluster:
+  children:
+    k8s_master:
+      hosts:
+        10.240.0.11
+    k8s_workers:
+      hosts:
+        10.240.0.21
+        10.240.0.69
+        10.240.0.104
+      children:
+        preemptive_node:
+          hosts:
+            10.240.0.20
```

Comment:
Hi @123MwanjeMike is it possible to store these ip address in environment variables
e.g https://www.digitalocean.com/community/tutorials/how-to-create-a-kubernetes-cluster-using-kubeadm-on-ubuntu-16-04

Reply:
Hi @NoahNsimbe 
Please provide more context on why we would need to do that.
Note that this is an inventory file using prod k8s hosts' internal IPs for the VPC network.

Reply:
thanks @123MwanjeMike for the clarification. I hadn't noticed that its an inventory file.

Reply:
No worries @NoahNsimbe 
I shifted from .ini style to .yaml format due to ease of readability